cnab/.gitignore

```diff
@@ -0,0 +1,2 @@
+istio*
```

Comment:
Since the bundle wants these files in a `charts` dir anyway, we don't need to ignore individual files if they are put into the common directory.

```suggestion
/charts
```

Reply:
done!